src/components/Pages/Service.js

```diff
@@ -2,6 +2,7 @@ import React from 'react';
 import { withRouteData } from 'react-static';
 import { injectIntl } from 'react-intl';
 import { findKey } from 'lodash';
+import path from 'path';
```

Comment:
took a quick look at what this module does. seems like its for normalizing file paths between OSes? Is that right? Just because I want to learn, why do we need it here?

Reply:
I'm using it to get the filename/extension of the image, as opposed to playing with regexes. See lines [48](https://github.com/cityofaustin/janis/pull/240/files/8074f4fb9743151ee2c75f8d6cd02cd2b2f5b6bc#diff-514450c133fd077285602d0f3a45b303R48), [49](https://github.com/cityofaustin/janis/pull/240/files/8074f4fb9743151ee2c75f8d6cd02cd2b2f5b6bc#diff-514450c133fd077285602d0f3a45b303R49)